modules/web/src/assets/config/changelog.json

```diff
@@ -1,9 +1,9 @@
 {
-  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.25.md#v22510",
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.25.md#v22511",
   "entries": [
     {
       "category": "fixed",
-      "description": "Fix Cilium application not upgrading properly due to invalid values"
+      "description": "Fix reconciling loop whenn resetting Application values to an empty value"
```

Comment:
https://github.com/kubermatic/kubermatic/pull/13741#issuecomment-2360764267

Reply:
Reviewing this PR because this might be urgent. Otherwise @Waseem826 should be able to take a look as well.

Reply:
Typo is fixed now.